src/components/share-modal/share-modal.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux';
+import get from 'lodash/get';
+
+import Component from './share-modal-component';
+
+const shareSocialMedia = [
+  {
+    link: `https://www.facebook.com/sharer/sharer.php?u=`,
+    className: 'facebookIcon',
+    alt: 'Facebook'
+  },
```

Comment:
I think, you can do similar thing with icons:
```
import facebookIcon from 'icons/facebook.png';
import twitterIcon from 'icons/twitter.png';
```
```suggestion
  icon: facebookIcon
},
```